app/views/idv/agreement/show.html.erb

```diff
@@ -14,9 +14,9 @@
       class: [
         'js-consent-form-alert',
         'margin-bottom-4',
-        flow_session[:error_message].blank? && 'display-none',
+        'display-none',
```

Comment:
If this is always hidden now, should we just remove the alert altogether? I don't see that there's anywhere that this alert is made visible.

Reply:
Removed component, ran feature spec, and that spec failed, so leaving it in for now.

Reply:
That test doesn't really make sense to me. I suspect it's "finding" the text because the non-JS driver doesn't understand the `display-none` is hiding the content with CSS, but in reality the message will never be visible to anyone.

Plus, we don't support no-JS for this flow, so doesn't seem that we should have any feature specs exercising the no-JS behavior IMO 🤷

Reply:
I agree with all that. Let's remove the component and specs in the followup PR on this ticket. I confirmed that the message doesn't show up in testing.